packages/library-legacy/src/connection.ts

```diff
@@ -960,7 +962,20 @@ export type SimulatedTransactionResponse = {
   accounts?: (SimulatedTransactionAccountInfo | null)[] | null;
   unitsConsumed?: number;
   returnData?: TransactionReturnData | null;
+  innerInstructions?: ParsedInnerInstruction[] | null;
 };
+const ParsedInstructionStruct = pick({
+  program: string(),
+  programId: PublicKeyFromString,
+  parsed: any(),
+});
+
+const PartiallyDecodedInstructionStruct = pick({
```

Comment:
Are there existing structs that we can reuse here? No worries if not, just wondering if we have the equivalent for getTransaction too. Not exactly sure how the structs and types are related!

Reply:
There is a [`ParsedInstructionResult` defined later on](https://github.com/solana-labs/solana-web3.js/blob/545c43992f555d54fe865788190e14eadd62264d/packages/library-legacy/src/connection.ts#L2225-L2229) that we could reuse — would just need to move it up, or we delete that and just use the new `ParsedInstructionStruct`